doc/build-instructions/Build_Instructions_V8.md

```diff
@@ -104,14 +104,36 @@ apt-get update \
    ...
 ```
 
-2. Download and setup **freemarker.jar** into a directory. The example commands use `/root` to be consistent with the Docker instructions. If you aren't using Docker, you probably want to store the **freemarker.jar** in your home directory.
+2. This build uses the same gcc and g++ compiler levels as OpenJDK, which might be
+backlevel compared with the versions you use on your system. Create links for
+the compilers with the following commands:
+```
+ln -s g++ /usr/bin/c++
+ln -s g++-7 /usr/bin/g++
+ln -s gcc /usr/bin/cc
+ln -s gcc-7 /usr/bin/gcc
+```
```

Comment:
If we're going to discuss this, I think it should explain how to get `gcc-7` and `g++-7` that are the targets of those links.

Reply:
This should be covered by the following, as the apt-get list includes `gcc-7 and g++-7`: `1. Install the list of dependencies that can be obtained with the `apt-get` command from the following section of the Dockerfile`.

Reply:
The section that was removed referred to gcc-4.8; the current requirement is for gcc-7.5. If you follow the steps listed in `mkdocker.sh --print`, you'll get gcc-7.5 and symbolic links are not required. I think this section should just stay removed.

Reply:
What about this part? `ENV CC=gcc-7 CXX=g++-7`
Are these exports required to use gcc 7 in the build in place of whatever else the user has installed?

Reply:
As a test, on Ubuntu 20 I did `apt install gcc` which installed gcc 9. Followed by `apt install gcc-7` but this doesn't replace the gcc 9 default you get when running `gcc --version`.

Reply:
If you use `apt install g++-7` then those environment variables will select that version (independent of whether you have other versions installed).

If you use `apt install g++` you get gcc-9 which worked last time I tried; those environment variables are not be needed in that case. On Ubuntu 18.04, you'll get version 7, so neither the symlinks nor the environment variables are necessary.

Reply:
I've pushed a new commit.

Reply:
I think it's better to tell the user to explicitly set the environment variables and use gcc 7. This is the compiler version we test and recommend. Otherwise we don't know what the use might have installed. Ub 16 installs gcc 5 by default which I don't think works.